### ISSUE TITLE
Take a closer look at the various MR_OUTLINE flags

### ISSUE THREAD

**Issue by Goober5000** [scp-fs2open/fs2open.github.com#819]
There are three:

```
#define MR_SHOW_OUTLINE				(1<<0)		// Draw the object in outline mode. Color specified by model_set_outline_color
#define MR_SHOW_OUTLINE_PRESET		(1<<14)		// Draw the object in outline mode. Color assumed to be set already.	
#define MR_SHOW_OUTLINE_HTL			(1<<26)		// Show outlines (wireframe view) using HTL method
```

If we're using HTL exclusively, the first and third flags should be merged.  After that's done, someone familiar with the model code should take a look at the `is_outlines_only_htl` and `is_outlines_only` variables.

I noticed this when reviewing some instances of `if (is_outlines_only_htl || (!is_outlines_only))` after the -nohtl removal in PR #641.  On first glance this looks like it devolves to `if (outlines || !outlines)` and then to `if (true)`, but it might be something more nuanced.

**Comment by MageKing17** [comment]
IIRC, `is_outlines_only` is still used when rendering a couple things (jump nodes and... FRED outlines, I think?), which (AFAIK) will not work when rendered "HTL-style", but, of course, you could simply try consolidating and seeing if anything breaks.

**Comment by Goober5000** [comment]
True, I do recall that jump nodes at least didn't work using the HTL method.
